runtime/src/in_mem_accounts_index.rs

```diff
@@ -106,10 +106,21 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
     }
 
     fn load_from_disk(&self, pubkey: &Pubkey) -> Option<(SlotList<T>, RefCount)> {
-        self.storage
-            .disk
-            .as_ref()
-            .and_then(|disk| disk.read_value(pubkey))
+        self.storage.disk.as_ref().and_then(|disk| {
+            let m = Measure::start("load_disk");
+            let entry_disk = disk.read_value(pubkey);
+            match &entry_disk {
+                Some(_) => {
+                    Self::update_time_stat(&self.stats().load_disk_us, m);
+                    Self::update_stat(&self.stats().load_disk, 1);
+                }
+                None => {
+                    Self::update_time_stat(&self.stats().load_disk_missing_us, m);
+                    Self::update_stat(&self.stats().load_disk_missing_us, 1);
```

Comment:
This one should be the count, not the timer, right?

Reply:
yep, good catch.